src/test/kotlin/com/workos/test/users/UsersTest.kt

```diff
@@ -480,4 +480,35 @@ class UsersTest : TestBase() {
 
     assertEquals("magic_auth_challenge_123", response.id)
   }
+
+  @Test
+  fun updateUserPasswordShouldReturnUpdatedUser() {
+    val workos = createWorkOSClient()
+
+    val userId = "user_123"
+    val newPassword = "new_password"
+
+    stubResponse(
+      "/users/$userId/password",
+      """{
+            "id": "$userId",
+            "email": "example@foo-corp.com",
+            "created_at": "2021-06-25T19:07:33.155Z",
+            "updated_at": "2021-06-25T19:07:33.155Z"
+        }""",
+      requestBody = """{
+            "userId": "$userId",
```

Comment:
Since the user ID is provided in the path we shouldn't need it in the body.

Reply:
this one shows my greeneess with Java, but I'm not exactly sure how to handle this without passing in `userId` to the body while still taking userId in the config. Have a direction to point me in?

Reply:
Looks like for other update methods we just pull the ID out into a separate argument https://github.com/workos/workos-kotlin/blob/main/src/main/kotlin/com/workos/organizations/OrganizationsApi.kt#L264-L267